ValeStyles/Couchbase/Latin.yml

```diff
@@ -0,0 +1,13 @@
+extends: substitution
+message: "Use '%s' instead of '%s'."
+link: https://docs.couchbase.com/styleguide/latin-abbreviations.html
+ignorecase: true
+level: error
+nonword: true
+action:
+  name: replace
+swap:
+  '(?i)\b(?:e\s*\.?\s*g\s*\.?)': for example
```

Comment:
will that also catch e.g. "egg" (because there's no \b after the 'g'?

Reply:
these regexes are quite hard to read.
Could we perhaps:
* skip the `\s*` before the dots, they're not really correct anyway
* skip the non-capturing `?:`
* make the options explicit 
* don't need to worry about the final ., as that'll trigger `\b` anyway
* consider more explicit alternatives, like `\b(eg|e[.] ?g)\b` where you at least see `eg` together at one point

Reply:
I was being a little lazy and just used Mr. Gippity for this. Try the cleaned up versions - "egg" will no longer be flagged.